src/components/signIn/SignIn.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PopupWithForm from '../popupWithForm/PopupWithForm';
+
+function SignIn(props) {
+  return(
+    <PopupWithForm name="sign-in" title="Sign in" isOpen={props.isOpen} onClose={props.onClose} isLoading={props.isLoading}>
+
+      <div className="popup__input-wrapper">
```

Comment:
could you have put the email and password sections in PopupWithForm, that way you'd only have to write them once?

Reply:
Good point! I've changed it